README.md

```diff
@@ -26,9 +26,14 @@ The rules for the `Veteran` and `Rescue Dog` roles (only used in the `Experience
 
 ## Limitations
 
-* A multiplayer (co-op) game can only be initiated by devices over the same network.
+* A multiplayer (co-op) game only works out of the box on devices over the same network. (including VPN)
 * Technical issues have been observed when playing on a macOS device.
 
-## Dependencies
-
-[pygame](https://github.com/pygame/pygame)
+## Setup
+### Prerequisites
```

Comment:
- [ ] We should add links on where they can install these from (It can be Googled of course, but we should provide as much information as we can).

- [ ] Maybe put these in a `requirements.txt` file and then run `pip install -r /path/to/requirements.txt` ([source](https://stackoverflow.com/a/15593865)) (Not sure if it's the right choice since there's a `pip` prerequisite itself and we would use `pip` for the `requirements` file...)

Reply:
requirements.txt is usually for if you want to install your project as a simple pip library to be used in another project. Since we're using distutils there no need for a requirements.txt since its already in the setup.py script. Also it wouldn't let us set up an entrypoint for the game. 

The prereqs here usually come with new installations of python anyway.

Reply:
Sounds good :)